packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1604,7 +1604,7 @@ abstract class ParentDataWidget<T extends ParentData> extends ProxyWidget {
   ///
   /// This is called just before [applyParentData] is invoked with the same
   /// [RenderObject] provided to that method.
-  bool debugIsValidRenderObject(RenderObject renderObject) {
```

Comment:
This is a public method, making it private is a breaking change. Do we need to rename this here?